docs/source/tutorial/queries.mdx

```diff
@@ -15,18 +17,25 @@ The `useQuery` hook leverages React's [Hooks API](https://reactjs.org/docs/hooks
 
 ## Fetching a list
 
+> **The code blocks below use TypeScript by default.** You can use the dropdown menu above a code block to switch to JavaScript.
```

Comment:
Since we're using this same bit of content at the top of each of the client tutorial pages, we could modularize it using MDX and import it in each so that we don't write the same thing 4 times.

The pattern we've been using for this is to create an MDX file in the **docs/shared** directory (or any sibling of **source**), paste the shared content in there, and then render it as if it were a JSX component.

_docs/shared/disclaimer.mdx_

```markdown
**The code blocks below use TypeScript by default.** You can use the dropdown menu above a code block to switch to JavaScript.

If you're using JavaScript, use `.js` and `.jsx` file extensions wherever `.ts` and `.tsx` are mentioned.
```

_docs/source/tutorial/queries.mdx_

```jsx
import Disclaimer from '../../shared/disclaimer.mdx';

<Disclaimer />
```